charts/observability-pipeline/README.md

```diff
@@ -13,7 +13,8 @@ Create a Kubernetes secret to store your API Key
 
 ```shell
 export HONEYCOMB_API_KEY=mykey
-kubectl create secret generic honeycomb --from-literal=api-key=$HONEYCOMB_API_KEY
+export HONEYCOMB_MGMT_API_SECRET=mymanagementkey
+kubectl create secret generic honeycomb-observability-pipeline --from-literal=api-key=$HONEYCOMB_API_KEY --from-literal=management-api-secret=$HONEYCOMB_MGMT_API_SECRET
```

Comment:
Should we do the same for the API key too?

Reply:
the honeycomb management api key id is not considered a secret so I made a section in the `.Values.controlPlane` to set it.

Reply:
The `HONEYCOMB_API_KEY` referenced here is used by the collectors and refinery